packages/orbit-components/src/Select/README.md

```diff
@@ -16,32 +16,31 @@ After adding import into your project you can use it simply like:
 
 Table below contains all types of the props available in the Select component.
 
-| Name            | Type                       | Default | Description                                                                                                                                                    |
-| :-------------- | :------------------------- | :------ | :------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| customValueText | `Translation`              |         | The custom text alternative of current value. [See Functional specs](#functional-specs)                                                                        |
```

Comment:
This is not to be removed. Just replaced by type string. The prop still exists and can still be used